index.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Startup Matchmaker</title>
+  <link rel="stylesheet" href="css/styles.css">
+</head>
+
+<body>
+  <div class="container">
+    <div class="header">
+      <header>
+        <h1>Startup Matchmaker</h1>
+        
+        <nav>
+            <div class="nav-item1">
+              <a href="#">Find Developers</a>
+            </div>
+
+            <div class="nav-item1">
+              <a href="#">Find Designer</a>
+            </div>
+
+            <div class="nav-item2">
+              <a href="#">How it Works</a>
+            </div>
+
+            <div class="nav-item2">
+              <a href="#">Our Team</a>
+            </div>
+
+            <div class="nav-item2">
+              <a href="#">Blog</a>
+            </div>
+        </nav>
+      </header>
+    </div>
+
+    <hr>
+
+    <div class="subheader">
+      <h2>Because two brains<div class="taste">taste</div><div class="are">are</div>better than one.</h2>
```

Comment:
to prevent you text from stacking change the `div` to `span`. Since spans are inline elements you text will not stack 🔨